src/main/kotlin/no/nav/medlemskap/regler/funksjoner/ArbeidsforholdFunksjoner.kt

```diff
@@ -9,6 +9,28 @@ import java.time.temporal.ChronoUnit
 import kotlin.math.abs
 
 object ArbeidsforholdFunksjoner {
+    private val enhetstyper = listOf("STAT", "FKF", "FYLK", "KF", "KOMM", "SF")
+
+    fun erArbeidsforholdetStatligEllerKommunalt(arbeidsforhold: List<Arbeidsforhold>, kontrollPeriode: Kontrollperiode): Boolean =
+        arbeidsforhold.erArbeidsgiverStatligEllerKommunal(kontrollPeriode)
+
+    infix fun List<Arbeidsforhold>.erArbeidsgiverStatligEllerKommunal(kontrollPeriode: Kontrollperiode): Boolean {
+        val kontrollPeriodeForSiste3Måneder = Kontrollperiode(kontrollPeriode.tom.minusMonths(3), kontrollPeriode.tom)
+
+        val erArbeidsforholdetStatlig =
```

Comment:
lur måte å løse det på med å lage en kontrollperiode på 3 mnd :) Tenker jeg ville bare hatt en return i stedet for å lage egen variabel her, slik vi har gjort i de fleste andre metoder